Ports/VVVVVV/patches/ReadMe.md

```diff
@@ -0,0 +1,39 @@
+# Patches for VVVVVV on SerenityOS
+
+**NOTE: Do not use the `dev` function of `.port_include.sh` when touching
+these patches!** The original source code uses CRLF line endings in the third
+party libraries, which are not preserved in the patches and `.port_include.sh`
+will throw a 'different line endings' error when applying. 
+
+Instead, unpack the tarball downloaded and create a repository by hand, then
+apply the patches with `git am`. This way, Git will detect the different line
+endings and convert the workdir to LF, which will allow the LF-ended patches
+to apply correctly, while internally CRLF will be preserved.
```

Comment:
The `dev` function doesn't really do anything else than what creating a repository by hand would do. That said, we can probably force `dev` to keep line endings unchanged, I will look into that tomorrow.

Reply:
Maybe resolved by #14303.